gallery/src/styles/Nivo/data.tsx

```diff
@@ -0,0 +1,335 @@
+/*
+   * Copyright (c) 2019-present Sonatype, Inc.
+   * This program and the accompanying materials are made available under
+   * the terms of the Eclipse Public License 2.0 which accompanies this
+   * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+   */
+
+export const rawBarData: [string, number][] = [
+  ['foxes', 10],
+  ['wolves', 34],
+  ['dogs', 1],
+  ['cats', 5],
+  ['penguins', 43],
+  ['chewbaccas', 89],
+  ['weasels', 64],
+  ['raccons', 12],
+  ['pandas', 98],
+  ['deers', 27],
+  ['bears', 32]
+];
+
+export const pieData = [
+  {
+    'id': 'donuts',
+    'label': 'donuts',
+    'value': 357
+  },
+  {
+    'id': 'ice cream',
+    'label': 'ice cream',
+    'value': 411
+  },
+  {
+    'id': 'pizza',
+    'label': 'pizza',
+    'value': 528
+  },
+  {
+    'id': 'fries',
+    'label': 'fries',
+    'value': 232
+  },
+  {
+    'id': 'oreos',
+    'label': 'oreos',
+    'value': 222
+  },
+  {
+    'id': 'quesadilla',
+    'label': 'quesadilla',
+    'value': 411
+  },
+  {
+    'id': 'nachos',
+    'label': 'nachos',
+    'value': 328
+  },
+  {
+    'id': 'salad',
+    'label': 'salad',
+    'value': 86
+  }
+];
+
+export const lineData = [
+  {
+    'id': 'japan',
+    'data': [
+      {
+        'x': 'plane',
+        'y': 105
+      },
```

Comment:
Don't be afraid to use more concise formatting such as 
```suggestion
      { 'x': 'plane', 'y': 105 },
```

Also is eslint passing with the quotes around the object keys? We usually don't quote those.

Reply:
> Don't be afraid to use more concise formatting such as
> 
> Also is eslint passing with the quotes around the object keys? We usually don't quote those.

Changed here: 001fe492aa0930e6eaf3daee018bf1bd91417b1d

Reply:
> > I don't have any errors from the quotes, but if I remove them I get errors.
> 
> I'm not seeing errors when I remove them

I removing the wrong ones:  d86dbadaa2512fccef72637ab11ba437e9fdc884